src/main/java/com/nike/cerberus/operation/core/WhitelistCidrForVpcAccessOperation.java

```diff
@@ -105,11 +108,11 @@ public void run(WhitelistCidrForVpcAccessCommand command) {
 
     @Override
     public boolean isRunnable(WhitelistCidrForVpcAccessCommand command) {
+        Regions region = command.getStackRegion().orElse(configStore.getPrimaryRegion());
         try {
-            cloudFormationService.getStackId(configStore.getPrimaryRegion(), Stack.IAM_ROLES.getFullName(environmentName));
+            cloudFormationService.getStackId(region, Stack.SECURITY_GROUPS.getFullName(environmentName));
         } catch (IllegalArgumentException iae) {
```

Comment:
In another exception above (https://github.com/Nike-Inc/cerberus-lifecycle-cli/pull/134/files#diff-35cf96ae7b57f8594ef82007bac0c7e9R126) IAE was changed to a general Exception for the same call, getStackID(). Should the same happen here?

Reply:
I think so, the underlying exception type changed in the AWS ask at some point.

- [ ] update exception type.